packages/react/src/primitives/types/collection.ts

```diff
@@ -50,6 +51,12 @@ export interface CollectionWrapperProps extends BaseStyleProps {
    * Search field placeholder
    */
   searchPlaceholder?: string;
+
+  /**
+   * @description
+   * Controls what is displayed when no results are found from the Collection search
+   */
+  noResultsFound?: React.ReactNode;
```

Comment:
I know I proposed this name in the original ticket, but what if we name this more consistently with the other search related props?
```suggestion
  searchNoResultsFound?: React.ReactNode;
```

Reply:
Good idea